pyperf/tests/__init__.py

```diff
@@ -27,6 +22,12 @@
 
 from pyperf._utils import popen_communicate, popen_killer
 
+if sys.version_info[0] == 2:
```

Comment:
I would prefer to reuse six.PY2 or six.PY3 for that.

Reply:
no worries.

I noticed a few other uses of `sys.version_info`, so used that.  Turns out there are only five, and two are replacable with six.PY2/PY3.